### DISCUSSION TITLE
Github Student developer pack verification

### DISCUSSION THREAD

**Discussion by gyanshupathak** [community/community#77790] [GitHub Education]
### Hi everyone,

Hi , I am student of IIT Roorkee ,India  and while i was signing up for Github student developer pack it said :You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

as i am in my home in semester break how it will work.
suggest me any solution for this!

**Comment by dakshithadissanayaka** [comment]
The error message you're encountering suggests that GitHub is unable to verify your location based on your academic affiliation. This is because your institution, IIT Roorkee, is not listed as having a specific campus location. Since you're currently at home during semester break, GitHub cannot confirm your proximity to the campus.

To resolve this issue, you can try the following options:

1. **Contact GitHub Education Support:** Reach out to GitHub Education support and explain the situation. They may be able to manually verify your enrollment status and approve your application based on the available information.

2. **Provide Additional Documentation:** If possible, gather any additional documentation that supports your enrollment at IIT Roorkee, such as official transcripts, student ID cards, or online enrollment confirmation emails. Share these documents with GitHub Education support to strengthen your case.

3. **Request Campus Location Update:** Contact IIT Roorkee's administration and inquire about updating their academic affiliation information on GitHub. This might involve providing relevant campus location details to GitHub.

4. **Temporary Location Permission:** Consider temporarily enabling location access for your browser and operating system when submitting your application. This might allow GitHub to verify your location and approve your application.

5. **Alternative Verification:** Explore alternative verification methods offered by GitHub Education, if any, that don't rely on location verification.

Remember to be patient and persistent in your efforts to resolve this issue. GitHub Education support will be able to guide you further if any additional steps are required.
